README.md

```diff
@@ -439,12 +488,14 @@ also add another task that triggers the deployment to all remote containers.
 
     remoteContainers.each { config ->
         task "deployRemote${config.name.capitalize()}"(type: com.bmuschko.gradle.cargo.tasks.remote.CargoDeployRemote) {
-            description = "Deploys WAR to remote Tomcat '${config.name}'."
-            containerId = 'tomcat7x'
-            hostname = config.hostname
-            port = config.port
-            username = config.username
-            password = config.password
+            cargo {
```

Comment:
Here you are setting the property values for the extension and not the task type. Setting extension property values vs. task type values are two different concept. Mixing them up in this sample only works accidentally and will be confusing to users.

My guess is that you are applying the `cargo` plugin? For this specific use case you should be applying the `cargo-base` plugin. Maybe that's not fully clear from the readme file. Can you provide me with the full build script?